src/styles/App.css

```diff
@@ -5,6 +5,7 @@
 
    --primary-color: #FF983B;
    --color-background: #EEEFF4;
+   --color-base: #FFFFFF;
```

Comment:
Se tu colocou o color-base aqui seria interessante refatorar o css das classes para utilizar o --color-base toda vez que for utilizar #FFF